### DISCUSSION TITLE
Klipper reports: ERROR

### DISCUSSION THREAD

**Discussion by asusdk** [bassamanator/Sovol-SV06-firmware#29] [Q&A]
Hi there ^^

hm.. i cant get it to work:
![image](https://user-images.githubusercontent.com/54312862/232074860-1d9457a8-0ef7-438f-8901-bc9e844567d5.png)

Received my sovolsv06 plus today 2 weeks early ^^  but as you can see if have some trouble get the config to work.

Klipper reports: ERROR
Unable to open temperature file '/sys/class/thermal/thermal_zone0/temp'

Does someone have a suggestion how to fix this ?

**Comment by asusdk** [comment]
Ok, i fixed it with commenting out a Pi-Specific Temperature Sensor in the misc-macros.cfg:
![image](https://user-images.githubusercontent.com/54312862/232077847-6944a8fe-452a-458e-807f-4d2a8a6a934f.png)

**Reply by bassamanator** [reply]
Was just about to say that right now. 

Any other issues?

**Reply by asusdk** [reply]
Right now, no issues anymore, started first time 1 minute ago, if i get new Problems ill let you know ^^

**Reply by bassamanator** [reply]
Great. I've only had confirmation from 1 user about this working. Once you get 1 print in, please confirm here that everything went well.

Thanks, and happy printing!

**Reply by asusdk** [reply]
Yes for sure, my plan was to create a Thread showing the first prints and so on, if that's ok for you

**Reply by bassamanator** [reply]
Great!
